src/js/components/Calendar/Calendar.js

```diff
@@ -172,6 +173,7 @@ const Calendar = forwardRef(
       firstDayOfWeek = 0,
       header,
       locale = 'en-US',
+      messages,
```

Comment:
be sure to document this and add it to the types

Reply:
good catch Ill add that